apps/user-office-backend/src/resolvers/CustomScalars.ts

```diff
@@ -1,42 +1,42 @@
-import { GraphQLScalarType, Kind } from 'graphql';
-
-export type AnswerType = number | string | Date | boolean | number[];
-
-const coerce = (value: AnswerType) => {
-  if (typeof value === 'number' && Number.isInteger(value)) {
-    return value;
-  }
-
-  if (typeof value === 'boolean') {
-    return Boolean(value);
-  }
-
-  if (Array.isArray(value)) {
-    return value;
-  }
-
-  if (value instanceof Date) {
-    return value;
-  }
-
-  return value;
-};
-
-export const IntStringDateBoolArray = new GraphQLScalarType({
-  name: 'IntStringDateBoolArray',
-  serialize: coerce,
-  parseValue: coerce,
-  parseLiteral(ast) {
-    if (ast.kind === Kind.INT) {
-      return coerce(parseInt(ast.value, 10));
-    }
-    if (ast.kind === Kind.STRING) {
-      return ast.value;
-    }
-    if (ast.kind === Kind.BOOLEAN) {
-      return ast.value;
-    }
-
-    return undefined;
-  },
-});
+import { GraphQLScalarType, Kind } from 'graphql';
+
+export type AnswerType = number | string | Date | boolean | number[] | unknown;

```

Comment:
Why is unknown needed here?

Reply:
It is needed after update of graphql. Most probably they changed the types and here is the result https://github.com/graphql/graphql-js/blob/main/src/type/definition.ts#L545. It is a bit disappointing because now the AnswerType is considered as unknown and the other types doesn't make much sense anymore.